src/utils/datetime_test.cljs

```diff
@@ -189,3 +191,47 @@
                                               "it"
                                               #'utils.datetime/medium-date-time-format)]
        (is (= (datetime/day-relative epoch) "01 gen 1970, 12:00:00 AM")))))
+
+(deftest format-mute-till-test
```

Comment:
Great work with the tests @ibrkhalil

Reply:
@ilmotta
I had an error with tests failing, Because I mistakenly hardcoded a date string and didn't make the string react to the current day.

Kindly take a look to see if there's something that needs adjusting.

Reply:
fd19b061418faf458f6218c933287e6ece097253